Src/FluentAssertions/CallerIdentification/HandlerResult.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace FluentAssertions.CallerIdentification
+{
+    public enum HandlerResult
+    {
+        InProgress,
+        Handled,
```

Comment:
🤔 I don't quite understand the difference between `Handled` and `Done`. Maybe the name isn't quite right, or some docs can help here.

Reply:
After coming back to this code again, I think maybe `GoToNextSymbol` makes a lot more sense.